engine/XcodeUnitTestEngine.php

```diff
@@ -83,27 +86,43 @@ protected function loadEnvironment() {
           $config_path),
         $ex);
     }
+    return $config;
+  }
+
+  protected function loadEnvironment() {
+    
+    $config = $this->readArcConfig();
```

Comment:
@hossamghareeb I don't think this variable is used anywhere. We can safely delete `readArcConfig`